TAF_EasyRest/TestFramework/Test/ClientTests.cs

```diff
@@ -1,65 +1,65 @@
-namespace TestFramework.Test
-{
-    [TestFixture]
-    public class ClientTests
-    {
-        public IWebDriver Chromedriver { get; private set; }
+//namespace TestFramework.Test
```

Comment:
We must not do this. It will be a big problem later. We have to make changes and build this branch before merge.